squarelet/organizations/forms.py

```diff
@@ -175,35 +176,42 @@ class Meta:
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        domains = self.instance.domains.values_list("domain", flat=True)
-        domain_list = ", ".join(f"<b> {d}</b>" for d in domains)
-        if domain_list:
-            self.fields["allow_auto_join"].help_text = _(
-                "<br> Allow users to join this organization without an invite "
-                "if one of their verified emails matches one of the organization's "
-                "email domains. This organization"
-                f" has the following email domains set: {domain_list}."
-                "<br>To modify this list, please contact our support team"
-                " at <b>info@muckrock.com<b/>."
-            )
-        else:
-            self.fields["allow_auto_join"].help_text = _(
-                "<br>Allow users to join this organization without requesting "
-                "an invite if one of their verified emails matches one of the "
-                " organization's email domains. This organization does not "
-                "currently have any email domains set. "
-                "To add one, please contact our support team at "
-                "<b>info@muckrock.com</b>."
-            )
+
+        if not self.instance.verified_journalist:
+            # Remove allow_auto_join if not a verified journalist
+            self.fields.pop("allow_auto_join", None)
+
+        if "allow_auto_join" in self.fields:
```

Comment:
Is there any reason it would not be present besides the line right above?  Should this just be one if statement?